proposals/containerd.adoc

```diff
@@ -0,0 +1,55 @@
+== containerd Proposal
+
+*Name of project:* containerd
+
+*Description*
+
+containerd is an industry-standard container runtime with an emphasis on simplicity, robustness and portability. It is available as a daemon for Linux and Windows, which can manage the complete container lifecycle of its host system: image transfer and storage, container execution and supervision, and low-level storage, etc..
+
+containerd is designed to be embedded into a larger system, rather than being used directly by developers or end-users. containerd will serve as a core container runtime for the CNCF ecosystem.
+
+*Sponsor / Advisor from TOC*: Brian Grant
+
+*Unique Identifier*: containerd
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/docker/containerd
```

Comment:
CNCF doesn't have a formal policy that it has to be like this, but currently all 7 of our projects live in their own top-level GitHub organization. We'd be happy to look at hosting additional projects that originated at Docker, but it's not clear to me that there's any big benefit of putting them under the same organization. (The only thing that comes to mind is not needing to manually recreate teams, assuming a team for containerd and a hypothetical new project were identical.)

My suggestion would be to specify: "Planning to move the repo to `https://github.com/containerd/containerd`." Then, if you take a few extra weeks to do so, or we agree on a different GitHub org, it's not a big deal. The only real gating item for a project to come into CNCF after getting 6 votes from the TOC is an executed trademark transfer agreement.